src/components/tl/mlx5/mcast/tl_mlx5_mcast_coll.c

```diff
@@ -50,23 +282,22 @@ ucc_status_t ucc_tl_mlx5_mcast_bcast_start(ucc_coll_task_t *coll_task)
 
 void ucc_tl_mlx5_mcast_collective_progress(ucc_coll_task_t *coll_task)
 {
-    ucc_tl_mlx5_task_t           *task      = ucc_derived_of(coll_task, ucc_tl_mlx5_task_t);
-    ucc_status_t                  status    = UCC_OK;
-    ucc_tl_mlx5_mcast_coll_req_t *req       = task->bcast_mcast.req_handle;
+    ucc_tl_mlx5_task_t           *task   = ucc_derived_of(coll_task, ucc_tl_mlx5_task_t);
+    ucc_status_t                  status = UCC_OK;
+    ucc_tl_mlx5_mcast_coll_req_t *req    = task->bcast_mcast.req_handle;
 
     if (req != NULL) {
```

Comment:
how is it possible that req is NULL here?

Reply:
but yeah, this must not happen at all.